packages/botonic-react/src/components/message.jsx

```diff
@@ -77,6 +86,22 @@ export const Message = props => {
     return isFromUser() ? webchatState.theme.brandColor : '#F1F0F0'
   }
 
+  const getFontColor = colors => {
+    if (isFromUser()) {
+      if (webchatState.theme.customUserMessages.color) {
```

Comment:
If `customUserMessages` is not defined, this line and the function will break!
Same in the return and the 'else' part

Reply:
done